packages/framework/src/Component/Cron/CronModuleFacade.php

```diff
@@ -217,11 +212,11 @@ public function getCronModuleByServiceId(string $serviceId): CronModule
      */
     public function findAllIndexedByServiceId(): array
     {
-        if ($this->allIndexedByServiceId === null) {
-            $this->allIndexedByServiceId = $this->cronModuleRepository->findAllIndexedByServiceId();
-        }
-
-        return $this->allIndexedByServiceId;
+        return $this->localCacheFacade->getOrSaveValue(
+            static::CRON_CACHE_KEY,
+            fn () => $this->cronModuleRepository->findAllIndexedByServiceId(),
+            static::CRON_CACHE_KEY,
+        );
```

Comment:
I feel that we are not using implemented namespace, to me it shoudl look like this:

```suggestion
        return $this->localCacheFacade->getOrSaveValue(
            static::CRON_NAMESPACE,
            fn () => $this->cronModuleRepository->findAllIndexedByServiceId(),
            static::CRON_CACHE_KEY,
        );
```

Reply:
Same for second occurence

Reply:
thx